src/routes/message.js

```diff
@@ -0,0 +1,81 @@
+const express = require('express');
+const mongo = require('mongodb');
+
+const Message = require('../schemas/message');
+
+const router = new express.Router;
+
+router.post('/',async (req, res) => {
+
+    const newMessage = new Message({
+        ...req.body
+    });
+    try{
+        await newMessage.save();
+        res.status(200).send(newMessage);
+    }catch(error){
+        res.status(400).send(error);
+    }
+
+})
+
+router.post('/insertMany', async(req,res) => {
+    const messages = req.body;
+    try {
+        const response = await Message.insertMany(messages);
+        res.status(200).send(response);
+    } catch (error) {
+        res.status(400).send(error);
+    }
+    
+})
+
+router.get('/get', async (req,res) => {
+    const id = req.body.id;
+    try {
+        const o_ID = await new mongo.ObjectID(id);
+        const message = await Message.findById(o_ID);
+         if(!message){
+             res.status(400).send('Error: message with corresponding ID does not exist');
```

Comment:
Wrong status code. 404 is for not found

Reply:
All of the status code will be change accordingly